lib/src/polyline.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:math' as math;
+
+import 'package:turf/helpers.dart';
+
+class Polyline {
+  static String _encode(current, previous, factor) {
+    current = _py2Round(current * factor);
+    previous = _py2Round(previous * factor);
+    var coordinate = current - previous;
+    coordinate <<= 1;
+    if (current - previous < 0) {
+      coordinate = ~coordinate;
+    }
+    var output = '';
+    while (coordinate >= 0x20) {
+      output += String.fromCharCode((0x20 | (coordinate & 0x1f)) + 63);
+      coordinate >>= 5;
+    }
+    output += String.fromCharCode(coordinate + 63);
+
+    return output;
+  }
+
+  static _py2Round(value) {
+    // Google's polyline algorithm uses the same rounding strategy as Python 2, which is different from JS for negative values
+    return (value.abs() + 0.5).floor() * (value >= 0 ? 1 : -1);
+  }
+
+  static List<Position> decode(String polyline, {int precision = 5}) {
```

Comment:
small doc comment here as well

Reply:
Resolved in https://github.com/dartclub/turf_dart/pull/96